docs/websockets.md

```diff
@@ -54,3 +54,6 @@ def asWebSocketStreamAlways[S](s: Streams[S])(p: s.Pipe[WebSocketFrame.Data[_],
 ```
 
 Using streaming websockets requires the backend to support the given streaming capability (see also [streaming](requests/streaming.md)). Streaming capabilities are described as implementations of `Streams[S]`, and are provided by backend implementations, e.g. `AkkaStreams` or `Fs2Streams[F]`.
+
+When working with streams of websocket frames keep in mind that a text payload maybe fragmented into multiple frames.
+STTP provides two useful methods (`fromTextPipe`, `fromTextPipeF`) for each backend to aggregate these fragments back into complete messages.
```

Comment:
`sttp` (lower case) & it would probably be good to mention where these functions are (in MonixWebSockets etc.), as finding thme might be non-trivial